### PR TITLE
Update deeplink for service request 100tb

### PR DESCRIPTION
**Pull request by salarson**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: migration/sharepoint-online-and-onedrive-migration-speed.md [hunk 1]
```diff
@@ -116,31 +116,17 @@ If you are planning to migrate over 100TB, please submit a support request follo
...
 
-1. Navigate to https://admin.microsoft.com
-
-2. Ensure you are using the new admin center preview.
-
-    >[!Note]
-    >If you are using the old Microsoft 365 admin center you can skip step 8 as the "Description" field will not exist.
-
-3. On the left nav pane, select **Support**, and then select **New Service Request**. 
+1. As an administrator , click the following link which will populate a help query in the admin center.
```
Comment (JohanFreelancer9) [inline migration/sharepoint-online-and-onedrive-migration-speed.md:119]:
```suggestion
1. As an administrator, select the following link which will populate a help query in the admin center. 
```

File: migration/sharepoint-online-and-onedrive-migration-speed.md [hunk 2]
```diff
@@ -116,31 +116,17 @@ If you are planning to migrate over 100TB, please submit a support request follo
...
-2. Ensure you are using the new admin center preview.
-
-    >[!Note]
-    >If you are using the old Microsoft 365 admin center you can skip step 8 as the "Description" field will not exist.
-
-3. On the left nav pane, select **Support**, and then select **New Service Request**. 
+1. As an administrator , click the following link which will populate a help query in the admin center. 
 
-    ![New service request](media/new-service-request.png)
+[SharePoint Migration over 100TB](https://admin.microsoft.com/AdminPortal/?searchSolutions=SharePoint%20Migration%20over%20100TB)
```
Comment (JohanFreelancer9) [inline migration/sharepoint-online-and-onedrive-migration-speed.md:121]:
```suggestion
[SharePoint Migration over 100 TB](https://admin.microsoft.com/AdminPortal/?searchSolutions=SharePoint%20Migration%20over%20100TB)
```

File: migration/sharepoint-online-and-onedrive-migration-speed.md [hunk 3]
```diff
@@ -116,31 +116,17 @@ If you are planning to migrate over 100TB, please submit a support request follo
...
 
-    This will activate the **Need Help?** pane on the right-hand side of your screen.
+2. At the bottom of the pane, select **Contact Support**, and then select **New Service Request**. 
 
-4.  In the **Briefly describe your issue** area, enter "SharePoint Migration over 100TB".</br>
+3. Under **Description**, enter "SharePoint Migration over 100TB".
```
Comment (JohanFreelancer9) [inline migration/sharepoint-online-and-onedrive-migration-speed.md:125]:
```suggestion
3. Under **Description**, enter "SharePoint Migration over 100 TB". 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @salarson, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@JoanneHendrickson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**